src/auth/screens/events.screen.js

```diff
@@ -29,52 +30,58 @@ const mapDispatchToProps = dispatch =>
     dispatch
   );
 
-const styles = StyleSheet.create({
-  descriptionContainer: {
-    justifyContent: 'center',
-    flex: 1,
-    marginLeft: 10,
-    color: colors.primaryDark,
-    ...fonts.fontPrimaryLight,
-  },
-  linkDescription: {
-    ...fonts.fontPrimarySemiBold,
-  },
-  linkBranchDescription: {
-    ...fonts.fontCode,
-  },
-  deletedLinkBranchDescription: {
-    color: colors.greyDarkest,
-    ...fonts.fontCode,
-  },
-  date: {
-    color: colors.greyDark,
-  },
-  subtitleContainer: {
-    paddingHorizontal: 15,
-    paddingBottom: 10,
-    borderBottomColor: colors.greyLight,
-    borderBottomWidth: 1,
-  },
-  subtitle: {
-    color: colors.greyDark,
-    fontSize: normalize(11),
-    marginTop: 1,
-    ...fonts.fontPrimarySemiBold,
-    marginLeft: 39,
-  },
-  textContainer: {
-    flex: 1,
-    alignItems: 'center',
-    justifyContent: 'center',
-    marginHorizontal: 15,
-  },
-  noneTitle: {
-    fontSize: normalize(14),
-    textAlign: 'center',
-    ...fonts.fontPrimary,
-  },
-});
+const DescriptionContainer = styled.Text`
+  justify-content: center;
+  flex: 1;
+  margin-left: 10;
+  color: ${colors.primaryDark};
+  font-family: ${fonts.fontPrimaryLight.fontFamily};
```

Comment:
For styled components fonts, try to import from "src/config/styled-fonts.js".

Reply:
I used fonts because I remember seeing conflicting info on whether there was a plan to get rid of styledFonts on some PRs because it was duplicating the font listing... like a comment by @machour in #557 and the discussion in #533.

Reply:
@ZahraTee Yes, I see two kinds of codes. How about choosing any one but don't creating the 3rd one? 😄 
- `${fonts.fontPrimaryLight};`
- `font-family: ${styledFonts.fontPrimaryLight};`